dev/dags/dbt/jaffle_shop/packages.yml

```diff
@@ -0,0 +1,3 @@
+packages:
```

Comment:
By adding this, the previously passing tests will already start failing - unless the `dbt deps` code is introduced, right?

Reply:
Yeah it works when the dbt deps code is added but if that was removed but packages kept it would fail. Could create another project with a package? Or just ensure when we have the package we have the dbt deps up to you. 

The test deliberately sets the dbt_deps to False such that it would fail due to package error.

Reply:
It looks good, @DanMawdsleyBA , thank you!